### DISCUSSION TITLE
Overlay trails in rendered video

### DISCUSSION THREAD

**Discussion by roomrys** [talmolab/sleap#1030] [Ideas]
It would be nice to have trails overlaid on the rendered video.

originally posted [here](https://github.com/talmolab/sleap/issues/958#issuecomment-1252074551)

**Comment by gitttt-1234** [comment]
Closing this discussion here, opened a [new issue](https://github.com/talmolab/sleap-io/issues/428) in `sleap-io` for this feature request!
